app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,5 @@
+module AdminHelper
+  def category_options categories
+    [[t("option.root"), nil]] + categories.collect { |category| [category.name, category.id] }
```

Comment:
```suggestion
    categories.pluck :name, :id
```
sử dụng option include_blank: true thay cho [t("option.root"), nil]